src/handlers.rs

```diff
@@ -396,22 +357,21 @@ fn serialize_initgroups(groups: Vec<Gid>) -> Result<Vec<u8>> {
     Ok(result)
 }
 
-pub struct Host {
-    pub addresses: Vec<std::net::IpAddr>,
-    // aliases is unused so far
-    pub hostname: String,
-}
-
-impl Host {
+impl Hostent {
     fn serialize(&self) -> Result<Vec<u8>> {
         // Loop over all addresses.
         // Serialize them into a slice, which is used later in the payload.
         // Take note of the number of addresses (by AF).
         let mut num_v4 = 0;
         let mut num_v6 = 0;
         let mut buf_addrs = vec![];
+        let mut buf_aliases = vec![];
+        // Memory segment used to convey the size of the different
+        // aliases. The sizes are expressed in big endian encoded 32
```

Comment:
```suggestion
        // aliases. The sizes are expressed in native endian encoded 32
```

Note to myself: this is wrong.